tests/spark/test_spark_model_export.py

```diff
@@ -391,10 +393,39 @@ def test_sparkml_estimator_model_log(
 
 
 @pytest.mark.large
-def test_sparkml_model_log_invalid_args(spark_model_transformer, model_path):
-    # pylint: disable=unused-argument
-    with pytest.raises(MlflowException, match="Cannot serialize this model"):
-        sparkm.log_model(spark_model=spark_model_transformer.model, artifact_path="model0")
+@pytest.mark.parametrize("should_start_run", [False, True])
+@pytest.mark.parametrize("use_dfs_tmpdir", [False, True])
+def test_sparkml_transformer_model_log(
```

Comment:
Do we need this test? It looks like `test_transformer_model_export` does the same thing.

Reply:
I think the `start_run` and `dfs_tmpdir` behaviors are already covered in other tests.

Reply:
That makes sense, I'll delete this test